docs/tutorials/htcondor.md

```diff
@@ -0,0 +1,233 @@
+from one of the examples found in the `examples/` or `community/examples`
+directories.
+
+This tutorial will use community/examples/htcondor-pool.yaml, which provisions
+a basic auto-scaling HTCondor pool.
+
+* a new VPC network secured from the public internet
+* an HTCondor Access Point for users to submit jobs
+* an HTCondor Central Manager that will operate the pool
+* a Managed Instance Group to scale a pool of HTCondor Execute Points to serve
+  new jobs as they are submitted
+
+The blueprint community/examples/htcondor-pool.yaml should be open in the Cloud
+Shell Editor (on the left).
+
+This file describes the cluster you will deploy. After you have inspected the
+file, use the ghpc binary to create a deployment directory by running:
+
+```bash
+./ghpc create community/examples/htcondor-pool.yaml --vars "project_id=<walkthrough-project-id/>"
+```
+
+> **_NOTE:_** The `--vars` argument is used to override `project_id` in the
+> blueprint variables. The `--vars` argument supports comma-separated list of
+> name=value variables to override blueprint variables. This feature only
+> supports variables of string type.
+
+This will create a deployment directory named `htcondor-001/`, which
+contains the terraform needed to deploy your cluster.
+
+## Deploy the Cluster
+
+Use the following commands to run terraform and deploy your cluster.
+
+```bash
+terraform -chdir=htcondor-001/htcondor init
+terraform -chdir=htcondor-001/htcondor validate
+terraform -chdir=htcondor-001/htcondor apply -auto-approve
+```
+
+If you receive any errors during `apply`, you may re-run it to resolve them.
+The deployment will take about 3 minutes. There should be regular status updates
+in the terminal. If the `apply` is successful, a message similar to the
+following will be displayed:
+
+<!-- Note: Bash blocks give "copy to cloud shell" option.  -->
+<!-- "shell" or "text" is used in places where command should not be run in cloud shell. -->
+
+```shell
+Apply complete! Resources: xx added, 0 changed, 0 destroyed.
+```
+
+## Wait for the pool to be ready
+
+Once terraform has finished, you may SSH to the HTCondor Access Point:
+
+```bash
+gcloud compute ssh access-point-0 --tunnel-through-iap --project <walkthrough-project-id/> --zone us-central1-c
+```
+
+Alternatively, you may browse to the `access-point-0` VM and click on "SSH" in
+the Cloud Console at this address:
+
+```text
+https://console.cloud.google.com/compute?project=<walkthrough-project-id/>
+```
+
+Once you have command line access to the machine, you must wait for HTCondor and
+Docker to complete installation. This will take approximately 5 minutes from
+when `terraform apply` finished. When it is complete, a message will be printed
+to the screen:
+
+```text
+******* HTCondor system configuration complete ********
+```
+
+You should also verify that the pool is operational by executing
+
+```bash
+condor_status -schedd -autoformat Name
```

Comment:
- I waited until I saw `******* HTCondor system configuration complete ********`
- After printing that out the command prompt looked weird so I hit enter to get a fresh prompt
- I called `condor_status -schedd -autoformat Name` for the first time and got the following error:

```
Error: communication error
CEDAR:6001:Failed to connect to <10.0.0.2:9618>
```

I then called it again 30s later and got the expected output.

Reply:
Added common errors ("command not found", "Failed to connect") and advises waiting 5 minutes or more.

Reply:
Note for posterity: installation time can be slightly reduced by eliminating Docker from the equation but I feel there is value for having it on the access point if it is also available on the execute points.